source/includes/webhooks/_scorecard_deleted.md

```diff
@@ -0,0 +1,30 @@
+## Scorecard deleted
+
+This web hook only fires when individual scorecards are destroyed.  This occurs when the Harvest API delete endpoint is used, or when the delete scorecard link is used in the application. This will not fire when a candidate or application is deleted. A candidate or application being deleted implies all their scorecards have been deleted with them.
```

Comment:
This is going to sound crazy, but there are definitely cases where an application gets deleted and scorecards haven't been deleted. This might be due to the historical code/bugs, but we saw evidence of it when Tom shipped the "Add `application_id` to `/scorecards` in Harvest.

It might be worth scouring the GHR codebase to try and understand if scorecards really do get deleted when an application is.